yb-voyager/src/dbzm/config.go

```diff
@@ -269,8 +272,8 @@ func (c *Config) String() string {
 			c.SSLKey,
 			c.SSLRootCert)
 		conf = conf + sslConf
-		if c.ReplicationSlotName != "" {
-			conf = conf + fmt.Sprintf(postgresReplicationSlotNameTemplate, c.ReplicationSlotName)
+		if c.ReplicationSlotName != "" || c.PublicationName != "" {
```

Comment:
The "OR" check seems rather confusing. (If either is set, we are setting both in the config)
Maybe separate out both the configs?